@uportal/dashboard-carousel/src/components/Carousel.vue

```diff
@@ -20,6 +20,7 @@
           <div
             v-for="card of region.content"
             :key="card.name"
+            :style="{ width: Math.floor(100 / region.content.length) + '%' }"
```

Comment:
I'm not quite following this part, what is this length pulling from, and what is the percentage being used to do that flex doesn't?

Reply:
This is to set the items within each slide/page of the carousel to be equal width. I suppose I could use flex-basis for that?

Reply:
`flex-basis` could be a good solution.
If we can use pure CSS for sizing and layout that would be preferable.